fluent-langneg/README.md

```diff
@@ -0,0 +1,104 @@
+# fluent-langneg
+
+`fluent-langneg` is an API for negotiating languages. It's part of
+Project Fluent, a localization framework designed to unleash 
+the expressive power of the natural language.
+
+## Installation
+
+`fluent-langneg` can be used both on the client-side and the server-side.
+You can install it from the npm registry or use it as a standalone script.
+
+    npm install fluent-langneg
+
+
+## How to use
+
+```javascript
+import negotiateLanguages from 'fluent-langneg';
+
+const supportedLocales = negotiateLanguages(
+  requestedLocales, availableLocales, defaultLocale
+);
+```
+
+The API reference is available at
+http://projectfluent.io/fluent.js/fluent-langneg.
+
+## Strategies
+
+The API supports three negotiation strategies:
+
+* filtering (defualt)
```

Comment:
Also, please either use a `###` header or intend the text below by 4 space to make it part of the bullet point.